sites/cheerpx/src/content/docs/13-tutorials/setup-svelte.md

```diff
@@ -0,0 +1,126 @@
+---
+title: Svelte setup
+description: Setting up Svelte app
+---
+
+This tutorial will explain how to create a application with Svelte that works together with CheerpX.
+
+## Install Svelte
+
+First create a new Svelte app by running:
+
+```
+npm create svelte@latest my-app
+```
+
+1. Select the Skeleton Project option.
+2. Enable Typescript if needed.
+3. There are no additional options required for this tutorial.
+
+Next, navigate to your project directory and install the dependencies:
+
+```
+cd my-app
+npm install
+npm run dev
+```
+
+At this point, the development server will run from another terminal in the background.
+
+We're able to test the development server by visiting: http://localhost:5173/
+
+## Install Cheerpx
+
+Next, let's install CheerpX.
+
+This [link] will direct you to the installation page of CheerpX.
+
+For this to work all top level imports used by the CheerpX NPM module need to be enabled.
```

Comment:
For this to work top level awaits must be enabled by setting the build target to es2020. Top level awaits are by the CheerpX NPM.